packages/video_player/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.10.1+7
```

Comment:
Wouldn't this be at least a major update? Bumping to `1.2.0` could introduce breaking changes, and it's new features should at least a minor. Should go towards `0.11.0`